src/sequence/list/mod.rs

```diff
@@ -267,5 +276,63 @@ impl<'a, T> Iterator for IterArc<'a, T> {
 
 impl<'a, T> ExactSizeIterator for IterArc<'a, T> {}
 
+#[cfg(feature = "serde")]
+impl<T> Serialize for List<T> where T: Serialize {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let mut state = serializer.serialize_seq(Some(self.len()))?;
+        for item in self {
+            state.serialize_element(&item)?;
+        }
+        state.end()
+    }
+}
+
+#[cfg(feature = "serde")]
+impl<'de, T> Deserialize<'de> for List<T> where T: Deserialize<'de> {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
+        deserializer.deserialize_seq(ListVisitor{phantom: PhantomData})
+    }
+}
+
+#[cfg(feature = "serde")]
+struct ListVisitor<T> {
+    phantom: PhantomData<T>
+}
+
+#[cfg(feature = "serde")]
+impl<'de, T> Visitor<'de> for ListVisitor<T>
+where T: Deserialize<'de>,
```

Comment:
indent the where

Reply:
Will do. Do you also want me to change things like
```
impl<'de, K, V> Deserialize<'de> for RedBlackTreeMap<K, V> where
    K: Ord + Deserialize<'de>,
    V: Deserialize<'de> {
```
to put the `where` on the next line?

Reply:
where in the next line (that how I have been doing)